docs/request.response.structures.md

```diff
@@ -0,0 +1,179 @@
+# Expected incoming request and outgoing response datastructures for Presto models
+
+Presto models should use these structures to ensure that tools that consume the data can access in a consistant way across models
+
+Details of specific model's input and response formats are usually in the `<model_name>_response.py` files.  i.e. `classycat_response.py`
+
+## Incoming requests to presto
+
+General design
+
+* Models that don't handle requests in parallel need to implement batch format (but can give errors if they recieve more than 1 item)
```

Comment:
I think all models should support batch format, and those that can't do parallel ops just do them serially then return, IMO...